server/controller.js

```diff
@@ -0,0 +1,113 @@
+const express = require('express');
+
+const router = express.Router();
```

Comment:
I had to use express-promise-router to support async await routes. Does yours work without the express router? Perhaps the newer versions of node support this.

I'm not sure. In any case, if you run into problems, you can reference [this](https://node-postgres.com/guides/async-express#:~:text=Then%20I%20will%20install%20express%2Dpromise%2Drouter%20and%20use%20it%20to%20define%20my%20routes.%20Here%20is%20my%20routes/user.js%20file%3A) for the package I'm talking about. If you've tested this and it works, no need to change anything.